tests/testMeasureSources.py

```diff
@@ -34,6 +34,7 @@
 import numpy as np
 
 import lsst.pex.logging as pexLogging
+from lsst.log import Log
 import lsst.pex.exceptions
```

Comment:
`Log` is used in line 53 below, also added by this commit. 

However, I actually cannot find any logs in `afw` that logs to `afwDetection.Measure`.  It's likely for some old codes that have been replaced?  I will delete these debugging placeholders.